tests/Fixer/FunctionNotation/FunctionDeclarationFixerTest.php

```diff
@@ -161,6 +171,141 @@ function foo() /* bar */
      * @param string      $expected
      * @param null|string $input
      *
+     * @dataProvider provideClosureFunctionSpacingNoneCases
+     */
+    public function testFixClosureFunctionSpacingNone($expected, $input = null)
```

Comment:
How do you handle when `$expected === $input`? Can you link me to some test file that uses this?

As for the helper method,

```php
$defaultConfigCases = self::casesForConfigurations(array(null), array(
	array('a'),
	array('expected b', 'b'),
));

$spacingNoneCases = self::casesForConfigurations(array($spacingNone), array(
	array('expected a', 'a'),
	array('b'),
));

$sharedCases = self::casesForConfigurations(array(null, $spacingNone), array(
	array('c'),
	array('expected d', 'd'),
));

return $defaultConfigCases + $spacingNoneCases + $sharedCases;
```

looks much more cleaner than

```php
return array(
	array('a'),
	array('expected b', 'b'),
	array('expected a', 'a'),
	array('b'),
	array('c'),
	array('expected d', 'd'),
	array('c', 'c', $spacingNone),
	array('expected d', 'd', $spacingNone),
));
```

especially when there are more than two shared cases.